package.json

```diff
@@ -10,7 +10,7 @@
   "license": "",
   "description": "Perf Widget front-end and back-end",
   "engines": {
-    "node": "5.x.x"
+    "node": "5.6.0"
```

Comment:
I think this may be better to leave with `5.x.x` as we can get the upgrades for free by redeploying the service on Heroku.